Analysis/DataModel/include/AnalysisDataModel/ReducedInfoTables.h

```diff
@@ -201,6 +201,20 @@ using ReducedMuon = ReducedMuons::iterator;
 using ReducedMuonExtra = ReducedMuonsExtra::iterator;
 using ReducedMuonCov = ReducedMuonsCov::iterator;
 using Dilepton = Dileptons::iterator;
+
+namespace v0bits
+{
+DECLARE_SOA_COLUMN(PIDBit, pidbit, uint8_t); //!
+} // namespace v0bits
+
+// bit information for particle species.
+DECLARE_SOA_TABLE(V0Bits, "AOD", "V0BITS", //!
+                  //o2::soa::Index<>, v0bits::PIDBit);
```

Comment:
should be removed

Reply:
It is necessary. Then, Where do I have to define V0Bits?

Reply:
I meant just the commented line, not the table column. It is fine now